pkg/genericapiserver/config.go

```diff
@@ -254,14 +255,42 @@ func (c *Config) ApplyOptions(options *options.ServerRunOptions) *Config {
 				},
 			},
 			SNICerts: []NamedCertKey{},
-			ClientCA: options.ClientCAFile,
 		}
 		if options.TLSCertFile == "" && options.TLSPrivateKeyFile == "" {
 			secureServingInfo.ServerCert.Generate = true
 			secureServingInfo.ServerCert.CertFile = path.Join(options.CertDirectory, "apiserver.crt")
 			secureServingInfo.ServerCert.KeyFile = path.Join(options.CertDirectory, "apiserver.key")
 		}
 
+		if len(options.ClientCAFile) > 0 {
+			clientCAs, err := certutil.CertsFromFile(options.ClientCAFile)
+			if err != nil {
+				// normally this would be no-no, but its the minimal change to backport to 1.5 and
+				// every caller is going to do this.
+				panic(fmt.Errorf("unable to load client CA file: %v", err))
```

Comment:
can you at least add this same check into `ValidateRunOptions` as well so they get a nice error under normal circumstances?